ocs_ci/ocs/node.py

```diff
@@ -201,8 +202,8 @@ def drain_nodes(node_names):
     log.info(f"Draining nodes {node_names_str}")
     try:
         ocp.exec_oc_cmd(
-            f"adm drain {node_names_str} --force=true --ignore-daemonsets "
-            f"--delete-local-data",
+            f"adm drain {node_names_str} --force={force} --ignore-daemonsets "
+            f"--delete-emptydir-data",
```

Comment:
this has to be tested with OCP 4.6,4.7,4.8 to make sure this param is aviabale

Reply:
tested with 4.8, and will do on 4.6 and 4.7